8_september/ts_assignment/src/eight.ts

```diff
@@ -0,0 +1,21 @@
+function Pangram(str:string){
+    let alfa='abcdefghijklmnopqrstuvwxyz';
```

Comment:
why u have src and script folder with same code

Reply:
Script folder is auto generated by TypeScript Compiler